### PR TITLE
Update documentation to reflect how to gather Patroni diagnostics

### PR DESCRIPTION
**Pull request by barthisrael**
This commit changes the Lasso documentation so it teaches how Lasso gathers information from Patroni, and how the user can specify which Patroni configuration Lasso should take into consideration.

References: DC-590.

## What Changed?

Added information about Patroni support in Lasso.

### CODE REVIEW

**Review by jakubwartakEDB — APPROVED** [review]
LGTM!

File: product_docs/docs/lasso/4/describe.mdx [hunk 1]
```diff
@@ -624,6 +624,56 @@ Tuned status and profiles.
...
+
+  * file `/tools/patroni/cli/topology.out`: Output of `patronictl -c <conf_file> topology` command
+  * file `/tools/patroni/cli/show_config.out`: Output of `patronictl -c <conf_file> show-config` command
+  * file `/tools/patroni/cli/list.out`: Output of `patronictl -c <conf_file> list -e -f json` command
+  * file `/tools/patroni/cli/history.out`: Output of `patronictl -c <conf_file> history` command
+
+**Depth:** surface
+
+
+**Security Impact:** *low* &mdash;
```
Comment (djw-m) [inline product_docs/docs/lasso/4/describe.mdx:643]: To match the rest of the doc:

```suggestion
**Security Impact:** Low &mdash; 
```

File: product_docs/docs/lasso/4/describe.mdx [hunk 2]
```diff
@@ -624,6 +624,56 @@ Tuned status and profiles.
...
+Gathers Patroni configuration file that is found in the server
+
+Report output:
+
+  * file `/tools/patroni/config/<basename>`: Patroni configuration file
+
+**Depth:** surface
+
+
+**Security Impact:** *low* &mdash;
```
Comment (djw-m) [inline product_docs/docs/lasso/4/describe.mdx:657]:
```suggestion
**Security Impact:** Low &mdash; 
```

File: product_docs/docs/lasso/4/describe.mdx [hunk 3]
```diff
@@ -624,6 +624,56 @@ Tuned status and profiles.
...
+
+Report output:
+
+  * file `/tools/patroni/systemd/<service_name>_cat.data`: Output of 'systemctl cat <service_name>
+  * file `/tools/patroni/systemd/<service_name>_status.data`: Output of 'systemctl status <service_name>
+
+**Depth:** surface
+
+
+**Security Impact:** *low* &mdash;
```
Comment (djw-m) [inline product_docs/docs/lasso/4/describe.mdx:674]:
```suggestion
**Security Impact:** Low &mdash; 
```

**Review by djw-m — APPROVED** [review]
Couple of format tweaks which I will merge, otherwise LGTM

**Comment by djw-m** [comment]
Only merge when the next release of Lasso is made

**Comment by gvasquezvargas** [comment]
@barthisrael  and @djw-m  
I see there was a Lasso release on Sept 12. Does it mean this can go out?

**Comment by gvasquezvargas** [comment]
Synced with @wind39 and the contents of this PR are already covered by https://github.com/EnterpriseDB/docs/pull/6052/ so closing this one.

**Pull request closed without merging**